app/src/test/java/com/example/benchproject/data/movie/details/repo/MovieDetailsDataRepositoryTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.benchproject.data.movie.details.repo
+
+import com.example.benchproject.data.movie.details.entity.GenresRemote
+import com.example.benchproject.data.movie.details.entity.MovieDetailsRemote
+import com.example.benchproject.data.movies.common.api.MoviesApi
+import com.example.benchproject.domain.movie.details.entity.GenresModel
+import com.example.benchproject.domain.movie.details.entity.MovieDetailsModel
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import org.amshove.kluent.shouldBe
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+
+internal class MovieDetailsDataRepositoryTest {
+
+    private val movieDetailsRemote = MovieDetailsRemote(
+        54431,
+        "/something1Background.com",
+        "Test1",
+        listOf(
+            GenresRemote(1, "name1"),
+            GenresRemote(2, "name2"),
+            GenresRemote(3, "name3")
+        ),
+        "Example description",
+        2000000,
+        5.4,
+        "2022-02-10",
+        "/something1.com"
+    )
+    private val movieDetailsModel = Result.success(
+        MovieDetailsModel(
+            54431,
+            "https://image.tmdb.org/t/p/w500/something1Background.com",
+            "Test1",
+            listOf(
+                GenresModel(1, "name1"),
+                GenresModel(2, "name2"),
+                GenresModel(3, "name3")
+            ),
+            "Example description",
+            2000000,
+            5.4,
+            "2022-02-10",
+            "https://image.tmdb.org/t/p/w500/something1.com"
+        )
+    )
+    private val apiService: MoviesApi = mockk()
+    private val tested = MovieDetailsDataRepository(apiService)
```

Comment:
It will be best to stick to a convention where `tested` is your last val in test class, this way it's more readable